### PR TITLE
Prac 12 login form frontend

### PR DESCRIPTION
**Pull request by ElenaDu**
## One Line Description
UI with basic functionality for Login Form with input fields validation. Fixed minor issues for the Registration form as well.
## Requirements
The login form enables users to log in using the email and password they set up during registration. It includes field validation and displays errors as needed. Refer to Figma for design details.

## Test Steps

1. Test the login process when all fields are correctly filled.
2. Verify login with incorrect password and/or email.
3. Check error messages for the input fields.

## UI/UX Outcome
Screen Shot 2024-04-01 at 1 44 54 PM
Screen Shot 2024-04-01 at 1 43 13 PM

### CODE REVIEW

**Review by Kremifany — APPROVED** [review]
Looks excellent!!!!

File: src/components/LoginForm.tsx [hunk 1]
```diff
@@ -23,79 +23,129 @@ const labelStyle = {
...
+                            direction="row"
+                            spacing={4}
+                            display="flex"
+                            alignItems="center"
+                            marginTop={4}
+                            marginBottom={4}
+                        >
+                            <Button
+                                type="submit"
+                                style={{
```
Comment (oxangyal) [inline src/components/LoginForm.tsx:118]: I would start using theme.ts, if you need help, pls let me know.

**Review by oxangyal — APPROVED** [review]
LGTM

File: src/components/LoginForm.tsx (hunk 1 above)
Comment (ElenaDu) [reply src/components/LoginForm.tsx:118]: Thank you so much for your recommendation. I agree that using theme.ts is more beneficial. I just went ahead and used the fastest method. I will certainly follow your advice in the future to tidy up my code.

**Pull request merged**